src/module/token/minting/MintableERC721.sol

```diff
@@ -366,13 +258,16 @@ contract MintableERC721 is Module, EIP712, BeforeMintCallbackERC721, OnTokenURIC
             }
             return;
         }
+        console.log("passes initial test");
```

Comment:
Cleanup

Reply:
addressed in this PR:
https://github.com/thirdweb-dev/modular-contracts/pull/145